packages/terra-markdown/__mocks__/github-markdown-css.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Allows components that require terra-markdown to be used in jest tests. The tests used
+ * to fail when jest tried to import the css file but this fixes that.
+ */
+module.export = {};
```

Comment:
Instead of mocking it here, I recommend we just move it to the jestconfig file located at the root of this monorepo, and use the identity-obj-proxy. https://github.com/cerner/terra-core/blob/master/package.json#L36